cypress/integration/explore.spec.js

```diff
@@ -75,4 +75,48 @@ context('Actions', () => {
       .children('input')
       .should('not.be.checked');
   });
+
+  //TODO use constants for parameters frmo other PR
+  it('adds and remove tags as a filters', () => {
+    cy.visit('/');
+    const tag1 = 'tag1';
+    const tag2 = 'tag2';
+    const tag3 = 'fjksf dlfsd';
+    const tag3_encoded = 'fjksf%20dlfsd';
+    cy.get('[data-testid=search-for-resource-input]').type(search);
+    cy.get('[data-testid=search-for-resource-submit]').click();
+    cy.get('[data-testid=expand-filtering-options]').click();
+    cy.get(`[data-testid=filter-tags-input] input`).type(tag1 + '{enter}');
+    cy.get(`[data-testid=filter-tags-input] input`).should('have.value', '');
+    cy.get(`[data-testid=filter-tags-input] input`).type(tag2 + '{enter}');
+    cy.get(`[data-testid=filter-tags-input] input`).type(tag3 + '{enter}');
+    cy.get(`[data-testid=filter-tag-chip-0]`).contains(tag1);
+    cy.get(`[data-testid=filter-tag-chip-1]`).contains(tag2);
+    cy.get(`[data-testid=filter-tag-container]`).contains(tag3);
+    cy.location().should((loc) => {
+      expect(loc.search).to.eq(`?query=${search}&tag=${tag1}&tag=${tag2}&tag=${tag3_encoded}`);
```

Comment:
Jeg har lagt inn at url parameter navnet legges inn vha konstanten som blir brukt for å generere url'en, slik at hvis vi endrer navnet på den så slipper vi å debugge hvorfor testene ikke lengre fungerer. F.eks "query" og "tag" streng biten jeg tenker på.

Reply:
yes. ventet bare på merging før jeg tok inn konstantene :-)